apps/storybook/src/TimePicker.stories.tsx

```diff
@@ -77,11 +76,74 @@ export const Basic: Story<TimePickerProps> = (args) => {
 Basic.args = {
   date: new Date(2021, 4, 11, 14, 55, 22),
   setFocusHour: true,
-  hourRenderer: (date: Date) =>
-    date.getHours().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  minuteRenderer: (date: Date) =>
-    date.getMinutes().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  secondRenderer: (date: Date) =>
-    date.getSeconds().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  meridiemRenderer: (meridiem: MeridiemType) => meridiem,
+};
```

Comment:
Can we maybe add `use12Hours` as an arg? Since we're using it in both stories, we could add it to the block on L17

```tsx
args: {
    use12Hours: false
  },
```

Reply:
What would adding it to the block on L17 do?

Reply:
If we add an args to `StoryName.args = {...}`, then that arg will be local to that story. But if we add args to the `args` property of the block on the top, then it appears in all stories by default.

```tsx
export default {
  ...,
  args: {argName: defaultValue},
  ...
} as Meta<TimePickerProps>;
```

Then if you want to change the value in a particular story, you can override it in `StoryName.args`. If you want to disable edit in a particular story, you can override the `control` (like this [comment](https://github.com/iTwin/iTwinUI-react/pull/844#discussion_r984561800)), or to hide it can write something like

```tsx
Basic.argTypes = {
  argName: {
    table: {
      disable: true,
    },
  },
};
```

Reply:
Ah, I see. Thanks for explaining!

Reply:
Added